fabric-transfer-api-v1/src/main/java/net/fabricmc/fabric/api/transfer/v1/fluid/FluidVariantAttributes.java

```diff
@@ -52,6 +56,13 @@ public static void register(Fluid fluid, FluidVariantAttributeHandler handler) {
 		}
 	}
 
+	/**
+	 * Enable blue- and red-colored names for water and lava respectively.
+	 */
+	public static void enableColoredVanillaFluidNames() {
+		coloredVanillaFluidNames = true;
+	}
```

Comment:
Is this kind of enable-only behaviour really in scope for Fabric API? It feels like it should be a config option at the very least, so modpacks could disable it if they want to.

Reply:
It exists the replace a mixin I have in MI, as the behavior is not achievable otherwise because fabric already defines a `FluidVariantRenderHandler`. Mods can make this configurable if they want to by not calling the function.